front/components/navigation/NavigationSidebar.tsx

```diff
@@ -110,7 +120,23 @@ export const NavigationSidebar = React.forwardRef<
         {subscription.paymentFailingSince && <SubscriptionPastDueBanner />}
         {nav.length > 1 && (
           <div className="pt-2">
-            <Tab tabs={navs} />
+            <Tabs value={currentTab?.id}>
+              <TabsList>
+                {navs.map((tab) => (
+                  <TabsTrigger
+                    key={tab.id}
+                    value={tab.id}
+                    label={tab.label}
+                    icon={tab.icon}
+                    onClick={() => {
+                      if (tab.href) {
+                        void router.push(tab.href);
```

Comment:
Would be nice to have the NextJS `Link` accessible through this component.

Reply:
Will do in a separate PR